src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskSchemalessTest.java

```diff
@@ -21,7 +21,7 @@ public class ClickHouseSinkTaskSchemalessTest {
     private static ClickHouseHelperClient chc = null;
 
     @BeforeAll
-    private static void setup() {
+    public static void setup() {
```

Comment:
public can be also removed according https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall